server/base/src/main/java/org/apache/accumulo/server/util/ChangeSecret.java

```diff
@@ -63,12 +66,15 @@ public static void main(String[] args) throws Exception {
     Instance inst = opts.getInstance();
     if (!verifyAccumuloIsDown(inst, opts.oldPass))
       System.exit(-1);
-    String instanceId = rewriteZooKeeperInstance(inst, opts.oldPass, opts.newPass);
-    updateHdfs(fs, inst, instanceId);
+    if (!verifyHdfsWritePermission(fs))
+      System.exit(-1);
```

Comment:
A message telling the user they don't have permissions? That is printed in the catch (AccessControlException ace) in verifyHdfsWritePermission().

Will fix the System.exits.